enterprise/server/util/pebble/pebble.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
 	"github.com/buildbuddy-io/buildbuddy/server/util/alert"
+	"github.com/buildbuddy-io/buildbuddy/server/util/protoutil"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 	"github.com/cockroachdb/pebble"
 	"google.golang.org/protobuf/proto"
```

Comment:
can we remove this?

It looks like no because we still use proto.Clone and proto.Size from this library. Could we add versions of those in pebbleutil and then remove the /proto import entirely to avoid accidentally mixing and matching the two libraries?

Reply:
I added the method/type alias in the package and rename it to proto.  https://github.com/buildbuddy-io/buildbuddy/pull/5570